src/components/cylc/Info.vue

```diff
@@ -0,0 +1,261 @@
+<!--
+Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+  <div class="c-info">
+    <!-- The task summary
+
+    * The "-40" comes from the GraphNode offset, see the GraphNode compoent.
+    * The height is "200" + "40".
+    * The "99999" is the maximum component width (because we don't know
+      how wide it will be until we render it).
+    * The "overflow-x: hidden" prevents this 99999 width from causing
+      horizontal scrolling.
+    * The "8em" is the height that this SVG will be scaled to fill.
+    -->
+    <div style="overflow-x: hidden;">
+      <svg
+        preserveAspectRatio="xMinYMin"
+        viewBox="-40 -40 99999 200"
```

Comment:
This is not so good for long task names on a narrow window.

Happy to bump to a follow-up issue however

![127 0 0 1_5173_](https://github.com/user-attachments/assets/4bcbc997-4385-4aeb-a798-ebd06d411d00)

Reply:
Follow up pls.

Reply:
I think this should be a follow up issue.

Reply:
Besides making the icon smaller, I don't think there is actually very much we can do about this. Word wrapping is not an option here.

However, https://github.com/cylc/cylc-ui/issues/1898 will add inputs to allow the user to manually enter the task & cycle similar to the log view. These will likely use smaller font so may display more of the task name, although the log view task input will also get cut off after a point.

Reply:
I don't think we should necessarily use the `GraphNode` component to display the task name and jobs, we can do this more bespoke for this view. But happy to accept the quick win here and punt this to either #1898 or a new issue

Reply:
Aw, I quite like the `GraphNode`, it gives easy access to the task and jobs in a familiar format.

Reply:
Haven't really thought through #1898 yet, we could lift-and-shift the task/workflow selectors from the log view as we need much the same thing here, e.g. this view should display both workflow and task metadata.